src/components/simpleReview/SimpleReviewButtons.tsx

```diff
@@ -0,0 +1,62 @@
+import CommentDialog from "../CommentDialog";
+import React, { useState } from "react";
+import { CollectionStatus } from "../../types";
+
+export interface SimpleReviewButtonsProps {
+  status: CollectionStatus;
+  approveChanges(): void;
+  declineChanges(text: string): void;
+  rollbackChanges(text: string): void;
+}
+
+export default function SimpleReviewButtons({
+  status,
+  approveChanges,
+  declineChanges,
+  rollbackChanges,
+}: SimpleReviewButtonsProps) {
+  const [dialog, setDialog] = useState<"reject" | "rollback" | "">("");
+  return (
+    <>
+      <div className="simple-review-buttons">
+        {status === "to-review" && (
+          <>
+            <button
+              className="btn btn-success"
+              onClick={() => approveChanges()}>
+              Approve
+            </button>{" "}
+            <button
+              className="btn btn-danger"
+              onClick={() => setDialog("reject")}>
+              Reject
+            </button>
+          </>
+        )}
+        {status === "work-in-progress" && (
+          <button
+            className="btn btn-danger"
+            onClick={() => setDialog("rollback")}>
+            Rollback
+          </button>
+        )}
+      </div>
+      {dialog === "reject" && (
+        <CommentDialog
+          description="Leave a comment for the editor:"
```

Comment:
We could add a sentence here in order to explicit that the changes won't be undone (?)